diagnostic.md

```diff
@@ -76,7 +86,7 @@ var groceryList = [
 
 What is the value of `groceryList[2]`?
 
-_Replace this text with your answer._
+groceryList[2] = 'a container of milk'
```

Comment:
groceryList[2] would be 'a stick of butter' because remember, array index's start at 0 and not 1